Sources/Classes/PlayerCore/TinyVideoPlayer.swift

```diff
@@ -927,21 +934,41 @@ public extension TinyVideoPlayer {
      */
     public func setupCommandCenterTriggers() {
         
+        performCleanupActions()
+        
         let commandCenter = MPRemoteCommandCenter.shared()
         
-        let seekForwardCommand = commandCenter.seekForwardCommand
-        seekForwardCommand.isEnabled = true
-        seekForwardCommand.addTarget { event -> MPRemoteCommandHandlerStatus in
-
-            guard let _ = self.playerItem else {
```

Comment:
I really like what you did here to reduce code redundancy.  👍 

However passing self as an extra parameter into the helper function looks a bit strange. I propose the following implementation for the helper function:

```
func enable(command: MPRemoteCommand, with action: @escaping () -> MPRemoteCommandHandlerStatus) {

    let wasEnabled = command.isEnabled
    command.isEnabled = true

    let target = command.addTarget { event -> MPRemoteCommandHandlerStatus in

        guard let _ = self.playerItem else {
            if #available(iOS 9.1, *) {
                return .noActionableNowPlayingItem
            } else {
                return .commandFailed
            }
        }

        return action()
    }

    cleanupActions.append {
        command.removeTarget(target)
        command.isEnabled = wasEnabled
    }
}
```
Then it can be used like this:
```
enable(command: commandCenter.seekForwardCommand) { [weak self] in
    guard let position = self?.playbackPosition else {
        return .commandFailed
    }

    self?.seekTo(position: position + TinyVideoPlayerDefaults.seekingInterval)

    return .success
}
```

What do you think? 😉